src/content/docs/apm/agents/net-agent/installation/install-net-agent-windows.mdx

```diff
@@ -258,4 +262,10 @@ We recommend setting the environment variables **for each application** you want
   [Here are a few ways you can set this environment variable per process](https://discuss.newrelic.com/t/setting-net-core-agent-environment-variables-per-process/157750), rather than system-wide.
 </Callout>
 
+### .NET Core and custom instrumentation [#custom-instrumentation]
+
+ASP.NET Core apps are automatically instrumented upon installation, but other .NET Core frameworks will require [custom instrumentation](/docs/agents/net-agent/custom-instrumentation/introduction-net-custom-instrumentation)
```

Comment:
maybe say 'most', with link to compatibility doc.

Reply:
https://docswebsitedevelop-nettroubleshootwindowsinstall.gtsb.io/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core/#app-frameworks